### DISCUSSION TITLE
Widgets básicos

### DISCUSSION THREAD

**Discussion by Guilherme-Santos1324** [Flutterando/forum#187] [Perguntas e Respostas]
![retangulos flutter](https://github.com/Flutterando/forum/assets/145623400/19400e49-1c75-451d-84ba-23915e9a874d)
To tentando replicar esses retangulos usando o flutter, mas meu codigo não rodando. Alguém pode replicar pra mim por favor, é pra um projeto pessoal.

**Comment by eduardoflorence** [comment]
@Guilherme-Santos1324 assim não conseguimos te ajudar. Você tem que mostrar o seu código e mensagem de erro.

**Comment by loulouizz** [comment]
Imagino que seja devido o uso de Columns e Rows em sequência. Quando um Column tem como filho um Row ou outro Column, e vice-versa, muitas vezes é preciso usar um widget Flex ou Expanded para dar a noção para o widget de quanto espaço ele pode ocupar. Quando não é feito isso, onde deveriam estar os widgets fica apenas vazio. Isso é algo que eu acredito que esteja acontecendo ao ver a imagem que você enviou, mas para ter certeza só vendo o código. Se puder upar o código também poderei ajudar melhor!
